website/docs/r/vapp_vm.html.markdown

```diff
@@ -61,21 +73,28 @@ resource "vcd_vapp_vm" "web2" {
 
 The following arguments are supported:
 
-* `vapp_name` - (Required) The vApp this VM should belong to.
-* `name` - (Required) A unique name for the vApp
 * `catalog_name` - (Required) The catalog name in which to find the given vApp Template
+* `name` - (Required) A unique name for the vApp
 * `template_name` - (Required) The name of the vApp Template to use
-* `memory` - (Optional) The amount of RAM (in MB) to allocate to the vApp
+* `vapp_name` - (Required) The vApp this VM should belong to.
+* `accept_all_eulas` - (Optional; *v2.0+*) Automatically accept EULA if OVA has it. Default is `true`
 * `cpus` - (Optional) The number of virtual CPUs to allocate to the vApp
 * `initscript` (Optional) A script to be run only on initial boot
-* `network_name` - (Optional) Name of the network this VM should connect to
-* `vapp_network_name` - (Optional; *v2.1+*) Name of the vApp network this VM should connect to
-* `ip` - (Optional) The IP to assign to this vApp. Must be an IP address or
-  one of dhcp, allocated or none. If given the address must be within the
-  `static_ip_pool` set for the network. If left blank, and the network has
-  `dhcp_pool` set with at least one available IP then this will be set with
-  DHCP.
-* `power_on` - (Optional) A boolean value stating if this vApp should be powered on. Default is `true`
-* `accept_all_eulas` - (Optional; *v2.0+*) Automatically accept EULA if OVA has it. Default is `true`
+* `ip` - (**Deprecated**) The IP to assign to this vApp. Must be an IP address or
+one of dhcp, allocated or none. If given the address must be within the
+`static_ip_pool` set for the network. If left blank, and the network has
+`dhcp_pool` set with at least one available IP then this will be set with
+DHCP. **Conflicts with** with networks.
+* `memory` - (Optional) The amount of RAM (in MB) to allocate to the vApp
+* `network_name` - (**Deprecated**) Name of the network this VM should connect to. **Conflicts with** with networks.
+* `networks` - (Optional; *v2.1+*) List of network interfaces to attach to vm. **Conflicts with**: networks, ip. **Note**: all params of this parameter and itself do force recreation of vms!
+  * `orgnetwork` (Required) Name of the network this VM should connect to.
+  * `ip` (Optional) One of: dhcp, allocated, none or an ip.
+  * `ip_address_allocation_mode` (Optional) One of: DHCP, POOL, MANUAL or NONE. Defaults to "POOL"
```

Comment:
Need more explanation what POOL and NONE really mean.

Reply:
Fixed.